osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -809,6 +809,98 @@ public override bool OnBackButton()
             return false;
         }
 
+        private void increaseSpeed()
+        {
+            var rateAdjustStates = ModSelect.AllAvailableMods.Where(pair => pair.Mod is ModRateAdjust);
+            var stateDoubleTime = ModSelect.AllAvailableMods.First(pair => pair.Mod is ModDoubleTime);
+            bool rateModActive = ModSelect.AllAvailableMods.Count(pair => pair.Mod is ModRateAdjust && pair.Active.Value) > 0;
```

Comment:
These should not be doing mod queries or mutations via `ModSelect`. Use `selectedMods` and `OsuGameBase.AvailableMods` instead.

Reply:
Done in [3fdbd73](https://github.com/ppy/osu/pull/28071/commits/3fdbd735ce063653bb92b7570df716e700a9b529)